Gemfile

```diff
@@ -0,0 +1,13 @@
+source 'https://rubygems.org'
+
+gem 'berkshelf', '~> 4.0'
+gem 'chefspec'
+gem 'chef-sugar'
+gem 'rubocop', '= 0.37.2'
+
+group :integration do
+  gem 'test-kitchen',    '~> 1.7.0'
+  gem 'kitchen-vagrant', '~> 0.19.0'
+  gem 'winrm-transport', '~> 1.0'
+  gem 'winrm-fs'
+end
```

Comment:
I reintroduced it because I couldn't get rubocop versions to line up correctly. I didn't know we removed it for a specific reason - I'm happy to remove it. (Also would like the history behind that for my own personal knowledge...)